book/website/community-handbook/content.md

```diff
@@ -0,0 +1,139 @@
+
+For example, the case study about the [Statistical Methods Manuscript](https://the-turing-way.netlify.app/reproducible-research/case-studies/statistical-methods-manuscript.html) in the Guide for Reproducible Research follows this file structure:
+
+```
+book/website
+│
+└───reproducible-research <---- (Folder for the Guide to Reproducible Research)
+│   │
+│   └───binderhub
+│   │
+│   └───case-studies <---- (folder containing subchapter for case studies the Guide to Reproducible Research)
+│   |   │   statistical-methods-manuscript.md <---- (Statistical Manuscripts Case Study)
+|   |   |   example-case-study.md
+|   |   |   ...
+|   |
+|   |   ...
+|   |   case-studies.md <---- (Landing page for Case Studies chapter)
+|   |   ...
+│   
+└───project-design <---- (Folder for the Guide for Project Design)
+    │   project-design.md
+    │   ...
+```
+
+More case studies (such as `example-case-study-file.md`) for the Guide for Reproducible Research can be added in the same location as `statistical-methods-manuscript.md`.
+
+### Chapters
+
+Chapters are sub-folders within Guides.
+
+For example, the [Version Control](https://the-turing-way.netlify.app/reproducible-research/vcs.html) chapter in the Guide for Reproducible Research has the following structure:
+
+```
+book\website
+│
+└───reproducible-research <---- (folder for the Guide to Reproducible Research)
+│   │   reproducible-research.md ---- (Guide's Landing Page)
+│   │   vcs.md <---- (Landing page for the Version Control chapter)
+│   │   example-chapter.md (landing page for a new chapter)
+|   |
+│   └───vcs (chapter folder)
+│   |   │   vcs-workflow.md
+|   |   |   vcs-git.md
+|   |   |   vcs-git-commit.md
+|   |   |   ...
+|   |   |   vcs-personal-stories.md
+│   |   │   vcs-checklist.md
+│   |   │   vcs-resources.md
+|   └───example-chapter (new chapter folder)
+│       │   example-subchapter.md
+|       |   ...
+|       |   example-personal-stories.md
+│       │   example-checklist.md
+│       │   example-resources.md
+│   
+└───project-design <---- (folder for the Guide for Project Design) 
+    │   project-design.md
+    │   ...
+```
+
+To add more chapters to the Guide for Reproducible Research, 
+use the file locations of `example-chapter`'s constituent files for reference.
+
+(ch-content-checklist)=
+## Contribution Checklist
+
+### Follow the style and consistency guidelines
+
+As you write your chapter, keep _The Turing Way's_ [style](https://the-turing-way.netlify.app/community-handbook/style.html) and [consistency](https://the-turing-way.netlify.app/community-handbook/consistency.html) recommendations in mind.
+This ensures that your new content is accessible, and fits the overall style, structure, and formatting of the book.
+
+### Name files/folders appropriately
+
+Please follow _The Turing Way's_ conventions for naming files.
+With proper file names, other contributors can easily identify the purpose of your files and add to or improve them if necessary. 
+
+
```

Comment:
Here you could place the checkpoint on adding new chapters/case studies (if entirely new ones were started with the templates) to the appropriate location in the directory.